cuttlefish-integration-gigabyte-arm64/README

```diff
@@ -0,0 +1,41 @@
+The Debian Package cuttlefish-integration-gigabyte-arm64
+----------------------------
+
+This is the metapackage for Google's Gigabyte Ampere project.
```

Comment:
There are multiple references to this being a "metapackage." The debian wiki defines a [metapackage](https://wiki.debian.org/metapackage) as

> Package that depends on a group of [packages](http://www.debian.org/distrib/packages) that relate to a common activity. Metapackages are empty and exists to link related packages together via dependencies.

This package is not empty as it includes `.install` and `.postinst` members that introduce files and make system changes directly.

Can this be called a "package" throughout rather than a "metapackage"?